src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -753,9 +754,9 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    */
 
   /**
-   * Callback signature used for a successful publish preflight.
+   * Callback signature used for a successful publish operation.
    */
-  using PreflightSuccessCallback = std::function<void(std::size_t, EntityReference)>;
+  using PublishSuccessCallback = std::function<void(std::size_t, EntityReference)>;
```

Comment:
[fix] Can we keep these as two specific callbacks (`PreflightSuccessCallback` and `RegisterSuccessCallback`). The entity hasn't been published at this point, so it feels a bit misleading to have a "publish success" before it has happened in `register`.

Reply:
Can we go with `PreflightSuccessCallback` and `RegisterSuccessCallback` for largely hand wavey/does what it says on the tine reasons.

Reply:
Split in d7346f5a